app/forms/conversion/involuntary/create_project_form.rb

```diff
@@ -12,7 +12,8 @@ def save
       regional_delivery_officer_id: user.id,
       task_list: Conversion::Involuntary::TaskList.new,
       directive_academy_order: directive_academy_order,
-      sponsor_trust_required: sponsor_trust_required
+      sponsor_trust_required: sponsor_trust_required,
```

Comment:
This is fine, but I would consider all involuntary code frozen from here on as we will strip it out soon.